lib/getNewReleases.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const fs = require('fs');
+const url = require('url');
+const { iterateOverFiles, getLatestVersion } = require('./home/helper');
+const { releases } = require('../releases');
+
+function getNewReleases() {
+    const jsonDirectory = path.join(process.cwd());
+    const docsPath = `${jsonDirectory}/docs`;
+    const basePath = `${url.pathToFileURL(jsonDirectory).toString()}/docs`;
+    const data = [];
+
+    iterateOverFiles([docsPath], basePath, data, getLatestVersion);
+
+    try {
+        data.forEach((item) => {
+            releases[item.platform] =
+                item.version === releases[item.platform].version
+                    ? { version: item.version, date: releases[item.platform].date }
+                    : { version: item.version, date: item.date };
+        });
+    } catch (e) {
+        console.log('Error in updating object', e);
+    }
+
+    console.log(releases);
```

Comment:
remove console.log 
or change console to debug

Reply:
Updated